src/openforms/contrib/zgw/clients/catalogi.py

```diff
@@ -325,6 +357,46 @@ def list_statustypen(self, zaaktype: str) -> list[dict]:
         results = response.json()["results"]
         return results
 
+    def get_all_role_types(
+        self,
+        *,
+        catalogus: str,
+        within_casetype: str,
+    ) -> Iterator[RoleType]:
+        params: RoleTypeListParams = {
+            "zaaktypeIdentificatie": within_casetype,
+        }
+        if self.allow_drafts:
+            params["status"] = "alles"
+
+        # get the case types so that we are filtering within the right catalogue, as
+        # the same case type identification may be defined in different catalogues
+        case_type_versions = (
+            self.find_case_types(
+                catalogus=catalogus,
+                identification=within_casetype,
+            )
+            or []
+        )
+        all_valid_roltype_urls: list[str] = sum(
+            (
+                case_type_version.get("roltypen", [])
+                for case_type_version in case_type_versions
+            ),
+            [],
+        )
+        if not all_valid_roltype_urls:
```

Comment:
Is there a reason to use `if not all_valid_roltype_urls` instead of `if len(all_valid_roltype_urls) == 0`?

Just for my own understanding :) (I'm reading some mixed preferences/reasons online. So i'm wondering what your reasoning is)

Reply:
it's considered the "Pythonic" way and relies on a value being falsy. In general, it's considered more readable, but I've seen some recent arguments/talks that this is actually going against the "explicit is better than implicit" guideline.

So in short - "this is how it's usually done in Python".

Reply:
Yeah exactly, i've read some similar messages about the "explicit is better than implicit", so i was wondering what is "best" for Open Forms.

Okay, check 👍